validators/products/create.js

```diff
@@ -5,13 +5,13 @@ const createValidator = Joi.object({
         .min(3)
         .max(30)
         .required(),
-    // .message("You must provide a name without num"),
     quantity: Joi.number().required(),
     price: Joi.number().required(),
     color: Joi.string().valid("red", "white", "blue", "orange", "black", "yellow", "green").required(),
     description: Joi.string().required(),
-    imageUrl: Joi.string().required(),
-    categories: Joi.string().valid("accesories", "building", "outfits").required()
+    imageUrl: Joi.string(),
+    cart: Joi.string(),
+    categories: Joi.string().valid("Accessories", "Building", "Outfits", "Please note that this item might be available based on the delivery area").required()
```

Comment:
what I the meaning of this field `Please note that this item might be available based ...`